en/embedding.html

```diff
@@ -60,9 +64,36 @@ <h2 id="provided-embedders">Provided embedders</h2>
 <a href="https://github.com/vespa-engine/vespa/blob/master/linguistics-components/src/main/resources/configdefinitions/language.sentencepiece.sentence-piece.def">the full configuration definition.</a>.
 </p>
 
-<p><p>The model(s) used must be supplied by the application package. You can find pre-trained models for
+<p>The model(s) used must be supplied by the application package. You can find pre-trained models for
 many languages <a href="https://github.com/bheinzerling/bpemb">here</a>.</p>
 
+<h3 id="wordpiece-embedder">WordPiece embedder</h3>
+
+<p>A native Java implementation of
+<a href="https://github.com/google-research/bert#tokenization">WordPiece</a>,
+which is commonly used with BERT models.</p>
+
+<p>To use the
+<a href="https://github.com/vespa-engine/vespa/blob/master/linguistics-components/src/main/java/com/yahoo/language/wordpiece/WordPieceEmbedder.java">WordPiece embedder</a>
+add it to your services.xml, e.g:</p>
+
+<pre class="code">
+&lt;component id=&quot;com.yahoo.language.weordpiece.WordPieceEmbedder&quot; bundle=&quot;linguistics-components&quot;&gt;
+    &lt;config name=&quot;language.wordpiece.word-piece&quot;&gt;
+        &lt;model&gt;
+            &lt;item&gt;
+                &lt;language&gt;unknown&lt;/language&gt;
+                &lt;path&gt;model/en.wiki.bpe.vs10000.model&lt;/path&gt;
```

Comment:
Should this be a pointer to the vocabulary file (.txt)?

Reply:
Yes, changed to a better example - thanks!